views/peeps/new.erb

```diff
@@ -0,0 +1,27 @@
+<body style="background-color:powderblue;">
+
+<img src="public/images/Chitter_logo.png" alt="Chitter Logo">
```

Comment:
Logo looks nice but I couldn't see it on my localhost. not sure if this is a Sinatra thing for images based on some comments I saw in the cohort chat